static/css/styles.css

```
@@ -583,6 +583,24 @@ h2 {
 */
 
 
+.other-products-heading {
+  margin: 40px 10px 10px 10px;
+  width: 94%;
+  font-weight: bold;
+  color: #55565a;
+  font-size: 18px;
+  padding-bottom: 10px;
+  border-bottom: 1px solid black;
```

No need to set the color on this. It's better to let the line color be inherited from the border color unless it's causing problems. That way if we were to ever change the color scheme, this would update as well. 

```suggestion
  border-bottom: 1px solid;
```